osu.Game/Database/EFToRealmMigrator.cs

```diff
@@ -209,6 +228,9 @@ private void beginMigration()
                 // If we were to not do this, the migration would run another time the next time the user starts the game.
                 deletePreRealmData();
 
+                // If something went wrong and the disposal token wasn't invoked above, ensure it is here.
+                realmBlockOperations?.Dispose();
```

Comment:
I had thought `realmBlockOperations` being non-null is if `RealmAccess.CreateBackup` throws, but that can never be the case because it's surrounded by a `using` which is essentially a `try-finally` code with object being disposed in `finally` scope.

Makes me wonder if there's ever a case where `realmBlockOperations` could be non-null, but doesn't seem harmful to have anyways.